snapshot-bump/snapshot-bump.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+function help {
+  echo 'usage: ./snapshot-bump.sh [args]'
+  echo " "
+  echo "Bumps some semver style version property to the next SNAPSHOT version."
+  echo " "
+  echo "--file, -f, env var: INPUT_FILE         the properties file that the version is in. default: gradle.properties"
+  echo "--prop, -p, env var: INPUT_PROPERTY     the version property that will be bumped. default: packageVersion"
+  echo ""
+  echo "--help, -h                              show this help blurb"
+  exit 0
+}
+
+ARGS=()
+
+while test $# -gt 0; do
+  case $1 in
+    -h|--help)
+      help
+      ;;
+    -f|--file)
+      shift
+      export INPUT_FILE=$1
+      shift
+      ;;
+    -p|--prop)
+      shift
+      INPUT_PROPERTY=$1
+      shift
+      ;;
+    *)
+      echo "unknown option $1"
+      shift
+      ;;
+  esac
+done
+
+if [ -z $INPUT_FILE ]; then
+  INPUT_FILE=gradle.properties
+  echo "using default file: gradle.properties"
+fi
+
+if [ -z $INPUT_PROPERTY ]; then
+  INPUT_PROPERTY=packageVersion
+  echo "using default property: packageVersion"
+fi
+
+if ! [[ `grep $INPUT_PROPERTY $INPUT_FILE` ]]; then
+  echo "property $INPUT_PROPERTY not found in file $INPUT_FILE. bailing"
+  help;
+fi
+
+VERSION_PROP=`grep packageVersion gradle.properties` # ex. packageVersion=1.7.8
```

Comment:
Should this be:
```
VERSION_PROP=`grep $INPUT_PROPERTY $INPUT_FILE`
```

Reply:
Yes! Thanks
https://github.com/mindjolt/actions/pull/2/commits/2654b01670ee4dbfdbc25c763b4d332a3e395172